### PR TITLE
modify dummy data with key change & extra records

### PR DESCRIPTION
**Pull request by kgaikwad**

### CODE REVIEW

File: ros/resources/hosts.py [hunk 1]
```diff
@@ -14,18 +14,18 @@ def get(self):
                     'id': '12345-57575757',
                     'account': '12345',
                     'vm_uuid': '12345a1',
-                    'state': 'Oversized',
-                    'recommendation_count': 4,
+                    'state': 'Crashloop',
```
Comment (apuntamb) [inline ros/resources/hosts.py:17]: Is `Oversized` no longer a state to be checked?

File: ros/resources/hosts.py [hunk 2]
```diff
@@ -14,18 +14,18 @@ def get(self):
...
                     'organization_id': 1,
                     'performance_score': {
-                        'cpu_score': 70,
+                        'cpu_score': 20,
                         'memory_score': 20,
-                        'io_score': 80
+                        'io_score': 20
                     },
                     'facts': {
                         'provider': 'AWS',
```
Comment (patilsuraj767) [inline ros/resources/hosts.py:26]: I think `cloud_provider` will be more appropriate key here. Also, on host inventory side key name is `cloud_provider`

**Review by patilsuraj767 — CHANGES_REQUESTED** [review]
LGTM :+1:  Just added one small point.

File: ros/resources/hosts.py (hunk 1 above)
Comment (kgaikwad) [reply ros/resources/hosts.py:17]: State value should be based on scoring, right? I have modified the scoring for this record.

**Review by patilsuraj767 — APPROVED** [review]

**Review by apuntamb — APPROVED** [review]
looks ready! :+1:

**Comment by apuntamb** [comment]
Merged, thanks @kgaikwad!

**Pull request merged**